### PR TITLE
Assessment view

### PR DESCRIPTION
**Pull request by ro-sa**
Please review assessment view page. merge conflicts already fixed.

### CODE REVIEW

File: src/views/ViewAssessment.vue [hunk 1]
```diff
@@ -5,14 +5,19 @@
...
     />
     <div class="d-flex flex-column justify-space-between card-container">
-      <a-breakdown-card class="card" :aBreakdown="aBreakdown" />
+      <a-breakdown-card
+        v-if="urlsLoaded && chartDataReady"
+        class="card"
+        :aBreakdown="aBreakdown"
+      />
       <material-breakdown-card
+        v-if="urlsLoaded && chartDataReady"
```
Comment (Exilliar) [inline src/views/ViewAssessment.vue:20]: can probably move the `v-if` into the div above and then remove the `v-if`'s on lines 20 and 15. Can be done as both these `v-if`'s are the same

**Review by Exilliar — COMMENTED** [review]
Looking good!

I think that you could update the ProjectInfoCard component to use the BECChip component so that the chip in that component shows the right color.

It also looks like the `ProjectCard` file has been duplicated. There's both one in the `components` folder and one in the `components/landing` folder. Either these are duplicated (so one should be removed). Or they are different but have the same name (so one should be renamed). I imagine this was caused by merging in from dev

Also, one small comment about a `v-if`

File: src/views/ViewAssessment.vue (hunk 1 above)
Comment (ro-sa) [reply src/views/ViewAssessment.vue]: thanks. code updated.

**Review by Exilliar — APPROVED** [review]
looks good!

**Pull request merged**